CONTRIBUTING.md

```diff
@@ -45,21 +45,26 @@ git clone https://github.com/<your-GitHub-username>/CopilotKit
 ## Step 3: Prepare the development environment
 
 ### 1)Install Prerequisites
+
 - Node.js 20.x or later
 - pnpm v9.x installed globally (npm i -g pnpm@^9)
 - Turborepo v2.x installed globally (npm i -g turbo@2)
 
 ### 2)Install Dependencies
+
 To install the dependencies using pnpm
 Go inside project folder and run :
 
 ```jsx
 pnpm install
 ```
+
 ### 3)Build Packages
+
 To make sure everything works, let’s build all packages once:
 
 ```jsx
+cd CopilotKit
```

Comment:
Ran into errors trying to build it, until I realized this must be done inside the CopilotKit directory, not in the repo root. So I think it's worth adding here.